connectors/google-maps-platform/element-templates/google-maps-platform-connector.json

```diff
@@ -0,0 +1,533 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Google Maps Platform Connector",
+  "id": "io.camunda.connectors.GoogleMapsPlatform.v1",
+  "description": "Orchestrate your Google Maps Platform operations with Camunda. You can validate and format address, get postal address, calculate distance, etc.",
```

Comment:
orhcestrate doesn't fit in this contect, add this as description:
The Google Maps connector validates addresses, retrieves postal addresses and calculates distances.